git-tar/function/ops.go

```diff
@@ -29,6 +33,21 @@ type cfg struct {
 	Frontend *string `json:"frontend,omitempty"`
 }
 
+type eventInfo struct {
```

Comment:
is this duplicated from buildshiprun?

Reply:
Yes, but I haven’t put anything in sdk. Tried to make it as simple as possible